### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by mforets** [comment]
done. for the style, i changed the TAB character by 4 spaces, so normally this affected all lines.. if didn't want this change let me know, and i'll revert :)

File: src/expmv.jl [hunk 1]
```diff
@@ -3,159 +3,161 @@ export expmv, expmv!
...
+            Base.A_mul_B!(p, A, vm[j])
+
+            for i=1:j
+                hm[i,j] = dot(vm[i], p)
+                # p[:] = p - hm[i,j]*vm[i]
+                p = axpy!(-hm[i,j], vm[i], p)
+            end
+            hm[j+1,j] = norm(p)
+
+	        if real(hm[j+1,j]) < btol # happy-breakdown
```
Comment (mforets) [inline src/expmv.jl]: this one is misaligned

**Comment by acroy** [comment]
Thanks! Looks good. The conflict is probably resolved automatically if you rebase against the current master (because I merged @garrison's PR yesterday).

**Comment by mforets** [comment]
Got it, thanks. Let's see if it applies now.

**Comment by acroy** [comment]
Perfect.

**Comment by mforets** [comment]
Ooops! The fix b8e02a was overwritten. How didn't i see it?! There were no conflicts when i rebased. Shall I cherry-pick that commit and push it on top of `mforets:refactoring`?

**Comment by acroy** [comment]
Don't worry. It was just one line and I just changed it manually.

**Pull request merged**